docs/rake_tasks.md

```diff
@@ -0,0 +1,102 @@
+# Rake Tasks
+
+This document provides further detail on the rake tasks provided as part of
+this repository.  It is intended to clarify usage and workflows for tasks whose
+complexity requires further explanation.
+
+## PE_XL
+
+The pe_xl task namespace is used for tasks that leverage the
+[pe_xl module](https://github.com/reidmv/reidmv-pe_xl/).
+
+As the namespace implies, the infrastructure managed by these task is not FOSS
+compatible and is intended only to be used for PE installations.
+
+
+### Deploy
+
+`pe_xl:deploy`
+: Clean; provision via ABS; run pe_xl plan.
+
+The `pe_xl:deploy` task is the provisioning task performed by performance rake
+tasks (i.e. `performance, `soak`, `autoscale`) when the `REF_ARCH` environment
```

Comment:
The ending ` is missing after "`performance" which prevents "`soak`" and "`autoscale`" from rendering properly.

Reply:
Thanks for catching this @billclaytor . I have updated